app/main.py

```diff
@@ -1,8 +1,19 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
The `people` dictionary is intended to store instances of `Person`, but currently, it stores the `Person` class itself. You should store the instance, like `Person.people[name.lower()] = self`.